tasks/src/zero-shot-image-classification/about.md

```diff
@@ -8,4 +8,23 @@ The data in this learning paradigm consists of
 
 - Seen data - images and their corresponding labels
 - Unseen data -  only labels and no images
-- auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
\ No newline at end of file
+- auxiliary information - additional information given to the model during training connecting the unseen and seen data. This can be in the form of textual description or word embeddings.
+
+
+## Use Cases
+
+### Autonomous Vehicles
+Autonomous Vehicles are a perfect use case for Zero-shot image classification. Occasionally in real-life situations, even we humans encounter new objects on the road that we are not sure how to react to instantly. Detecting + Classifying novel objects and deciding the correct actions to take are crucial for Robust Autonomous Vehicle Systems.
+
+### Image Retrieval
+Zero-shot learning resolves several challenges in existing Image retrieval systems. For example, with the rapid growth of unseen categories on the web, it is challenging to index images based on known categories.
+
+Zero-shot learning also has applicability in Image Instance Retrieval, where against an input image, images representing the same features are retrieved from a database. Conventional systems can only search for  semantically-related images, not always the exact instances.
+
+### Disease Classification
+Zero-shot learning can be helpful in disease classification, where we have a scarcity of labeled images for a specific disease. Here we can use a zero-shot image classifier that has learned from the dataset of other similar diseases along with the auxiliary information provided.
```

Comment:
I'm not very sure, @sayakpaul WDYT?

Reply:
I would not add this. Zero-shot predictions, if deemed unreliable, can wreak havoc on healthcare systems.